### ISSUE TITLE
Some Questions about the Dataset Part

### ISSUE THREAD

**Issue by WesternTrail** [guanyuezhen/AR-CDNet#2]
1) I see that the paths for the training set and validation set (Test) divided in your code are the same. What is the significance of doing so?
2) The path of Test2 points to the BCDD dataset, why use different types of datasets for validation instead of dividing it into parts from LEVIR-CD?
3) Can you further improve the Readme.md file and provide more information?

![image](https://github.com/guanyuezhen/AR-CDNet/assets/28615285/b17201f3-5ca0-4578-bd35-63b8350aa3c7)
